README.md

```diff
@@ -69,6 +69,16 @@ brew tap anchore/syft
 brew install syft
 ```
 
+**Container Image**
+
+Example of how to use Syft scratch image:
+
+
+```bash
+docker pull anchore/syft:latest
+docker run -v /var/run/docker.sock:/var/run/docker.sock anchore/syft -o json alpine:latest
```

Comment:
```suggestion
docker run -v /var/run/docker.sock:/var/run/docker.sock anchore/syft:latest -o json alpine:latest
```

Reply:
also, should we throw in `-t` to get the ETUI output? or leave as is for logger output? (open question)

Reply:
So I added the "-t" or Allocate a pseudo TTY option and the terminal hung, which is odd.

Reply:
gotcha, we'll leave this as is 👍  thanks for confirming